sdk/sdkservices/oauth.go

```diff
@@ -16,5 +16,5 @@ type OAuth interface {
 	Register(ctx context.Context, id string, cfg *oauth2.Config, opts map[string]string) error
 	Get(ctx context.Context, id string) (*oauth2.Config, map[string]string, error)
 	StartFlow(ctx context.Context, integration string, cid sdktypes.ConnectionID, origin string) (string, error)
-	Exchange(ctx context.Context, integration, code string) (*oauth2.Token, error)
+	Exchange(ctx context.Context, integration, code string, cid sdktypes.ConnectionID) (*oauth2.Token, error)
```

Comment:
A more appropriate location for `cid` - before `code`

Reply:
Help me understand the decision behind this for the future.

Reply:
No, it's because there's an implicit hierarchy: integration ID is the most general/basic detail to identify the entity we're working with, connection ID is a narrower focus than the integration type (single connection compared to multiple connections), and code is the most specific and ephemeral aspect of the connection init flow.